pysgrid/sgrid.py

```diff
@@ -252,6 +270,451 @@ def _save_common_components(self, nc_file):
                     dataset_grid_var.axes = ' '.join(axes)
         return grid_vars
 
+    def _get_grid_vars(self, name):
```

Comment:
@jay-hennen unless you plan to change this a lot don't make it a private method.